source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnRenderer.mm

```diff
@@ -82,6 +82,11 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
                                     multiplier:1.0
                                       constant:0].active = YES;
     }
+
+    column.hidden = !(elem->GetIsVisible());
+    NSString *hashkey = [NSString stringWithCString:elem->GetId().c_str() encoding:NSUTF8StringEncoding];
+    column.tag = hashkey.hash;
```

Comment:
These lines are repeated in a lot of places. Can we reduce the duplication? Put the logic in a base renderer class or something?